material_components_ios.bzl

```diff
@@ -46,3 +47,37 @@ def mdc_public_objc_library(
       includes = ["src"],
       enable_modules = 1,
       **kwargs)
+
+def ios_runners():
+  ios_test_runner(
+    name = "IPHONE_5_IN_8_1",
+    device_type = "iPhone 5",
+    os_version = "8.1",
+  )
+  ios_test_runner(
+    name = "IPAD_PRO_12_9_IN_9_3",
+    device_type = "iPad Pro (12.9-inch)",
+    os_version = "9.3",
+  )
+  ios_test_runner(
+    name = "IPHONE_7_PLUS_IN_10_3",
+    device_type = "iPhone 7 Plus",
+    os_version = "10.3",
+  )
+  ios_test_runner(
+    name = "DYNAMIC_RUNNER",
+    device_type = select({ ":xcode8_3_3": "iPad 2", ":xcode_9_0": "iPhone2017-C", "//conditions:default": "iPhone X" }),
+    os_version = select({ ":xcode8_3_3": "8.4", "//conditions:default": "11.0" }), 
+  )
+
+  native.config_setting(
+      name = "xcode8_3_3",
+      values = {"xcode_version": "8.3.3"},
+  )
+
+  native.config_setting(
+      name = "xcode_9_0",
+      values = {"xcode_version": "9.0"},
+  )
+
+  return [":IPHONE_5_IN_8_1", ":IPAD_PRO_12_9_IN_9_3", ":IPHONE_7_PLUS_IN_10_3", ":DYNAMIC_RUNNER"]
```

Comment:
Do all of the runners run on each test invocation? Or does blaze somehow identify which config to use based on the command line invocation?

Reply:
All the runners run on all the components, with the caveat of the DYNAMIC_RUNNER which identifies the argument passed in the blaze invocation (e.g. `blaze -xcode_version 9.1`), and then chooses based on the xcode_version we passed if to create an iPhoneX + iOS11.0 runner or an iPad2 + iOS8.4 for xcode version 8.3.3.

Reply:
Cool cool.